lib/redis/redis_keys.rb

```diff
@@ -33,4 +33,7 @@ module Redis::RedisKeys
   MESSAGE_SOURCE_KEY = 'MESSAGE_SOURCE_KEY::%<id>s'.freeze
   CUSTOM_FILTER_RECORDS_COUNT_KEY = 'CUSTOM_FILTER::%<account_id>d::%<user_id>d::%<filter_id>d'.freeze
   OPENAI_CONVERSATION_KEY = 'OPEN_AI_CONVERSATION_KEY::%<event_name>s::%<conversation_id>d::%<updated_at>d'.freeze
+
+  ## Sempahores / Locks
+  FACEBOOK_MESSAGE_CREATE_LOCK = 'FB_MESSAGE_CREATE_LOCK::%<sender_id>s::%<recipient_id>s'.freeze
```

Comment:
let's also add a comment here, something like  ` We don't want to process messages from the same sender concurrently  to prevent creating double conversations `
FACEBOOK_MESSAGE_MUTEX ?